src/providers/LocalVideoProvider/docs/useLocalVideo.stories.mdx

```diff
@@ -12,8 +17,13 @@ The `useLocalVideo` hook returns the video state of the local user, and a functi
   tileId: null | number;
 
   // Whether or not the local user has video enabled
+  // This does not tell whether the video preview from PreviewVideo component is enabled or not.
+  // This is only for the LocalVideo component rendered for the in-meeting screen.
```

Comment:
Should we backtick the component names here? Also, lets not say in-meeting screen as it can vary by context in which builders may use this. We can remove last line, the second should be enough I think.